components/form/RadioInput.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+// TODO: Formik is being imported but it not currently a dependency in package.json.
```

Comment:
@drewclem This seems like it was left over from pre-next? As far as I can tell, this component isn't used at all. Should we just delete it?

Reply:
Yeah, safe to remove altogether